### ISSUE TITLE
SDK Integration - Version previous 18

### ISSUE THREAD

**Issue by GaelCougnaud** [ezeeworld/B4S-Android-SDK#4]
Hi,

I'm trying to integrate your SDK (1.7.3 version) with Android Studio. Your SDK manifest declares minSDK to API Level 18. Our application have got a min SDK of 15. To use your SDK without changing our application compatiblities, we must use a tools:overridding for minSDK.
Of course, we don't initialize SDK or call it, if the APi Level of the device are lower than 18.
I did some quicks tests and that's seems to works (ie : no crash). 
But I want to be sure, no background code can happen with device with old API Level and induce crashes.

Best Regatds,
Gaêl

**Comment by eric-kok-ezeeworld** [comment]
Hi Gaêl,

You have correctly used the `tools:overrideLibrary` to enforce your lower SDK version. This can not do any harm, the SDK will enable/disabled features appropriately. For example, scanning for beacons is only available since Android 4.3 (and on hardware that support Bluetooth LE), but it can still run on older, unsupported devices without issue.

The full line of code to use is:
```
<uses-sdk tools:overrideLibrary="com.ezeeworld.b4s.android.sdk, com.ezeeworld.b4s.android.sdk.playservices" />
```

Regards,
Eric

**Comment by GaelCougnaud** [comment]
thanks for your prompt reply. I use the same code line in my manifest
